### DISCUSSION TITLE
Developer Help Needed - Load Source Code into new github account and deploy thru Vercel

### DISCUSSION THREAD

**Discussion by chrisrmor** [vercel/community#4708] [Help] [answered]
### Summary

We hired a marketing agency and they contracted a developer on their end to build a landing page for us. He built in vercel, felt like overkill but here we are. We have separated service from the agency, and thus lost access to the developer. He was kind enough to send the source code, and instructions to access, but I am not a developer (I'm a freaking CFO by trade) so I have no idea how to get the code up and running in my own github and link to vercel.

The landing page is working fine, but we have important changes that need to be made and we are completely hamstrung.

### Steps to Reproduce

See above.

**Comment by amyegan** [comment, marked as answer]
Hey @chrisrmor! I'm not sure whether you're trying to make changes yourself, or you're trying to hire a developer. I'm going to answer under the assumption that you are trying to make the change on your own.

If you already own the original project on Vercel and own the code on GitHub, you should be able to deploy changes pretty easily. You may need to [change the GitHub repository of the project](https://vercel.com/docs/deployments/git/vercel-for-github#changing-the-github-repository-of-a-project) if you are using a fork instead of the original repo. Then, deployments should happen automatically when changes are pushed to the GitHub repo.

Here are a few places in our documentation that provide more detail:
- Automatic deployment from GitHub repo: [Deploying GitHub Projects with Vercel](https://vercel.com/docs/deployments/git/vercel-for-github#)
- Other ways to deploy: [Deploying to Vercel](https://vercel.com/docs/deployments/overview#)
- Previews: [Preview Deployments Overview](https://vercel.com/docs/deployments/preview-deployments#)
